src/components/layout.css

```diff
@@ -620,3 +620,21 @@ pre tt:after {
     font-size: 100%;
   }
 }
+.w3-myfont {
```

Comment:
I love the this font for the nav-bar; but there's a way to add fonts at a central location without using CSS; we will use this approach; https://www.gatsbyjs.org/docs/typography-js/. I have a separate issue created (#21) for adding typography for the whole website so we will do it there if you don't mind. For now we'll do the nav-bar with default fonts but when we get to that issue we'll add this or a similar cursive font. :smile: